xfields/ibs/_equilibrium_emittance.py

```diff
@@ -183,37 +198,51 @@ def compute_emittance_evolution(
+        See the next parameter for possible scenarios and how this value is
+        used. Defaults to 0.
     emittance_constraint : str, optional
-        Can enforces constraints on the transverse emittance based on the
-        emittance coupling factor.
-        "Coupling" corresponds to the case where the
-        vertical emittance is the result of linear coupling.
-        "Excitation" corresponds to the case where the vertical emittance is
-        the result of an excitation (e.g. from a feedback system).
-        Default is "Coupling".
-    input_sigma_zeta : float
-        Used specified RMS momentum spread overwriting the natural one from
-        the `twiss` object. Default is None.
-    input_sigma_delta : float
-        Used specified RMS momentum spread overwriting the natural one from
-        the `twiss` object. Default is None.
-    natural_emittances : tuple of floats, optional
-        Natural emittances (horizontal, vertical, longitudinal).
-        If None, they are taken from the `twiss` object. Default is None.
+        If an accepted value is provided, enforces constraints on the transverse
+        emittances. Can be either "coupling" or "excitation", case-insensitively.
+        Defaults to "coupling".
+          - If `coupling`, vertical emittance is the result of linear coupling. In
+            this case both the vertical and horizontal emittances are altered and
+            determined based on the value of `emittance_coupling_factor` such that
+            the total transverse IS preserved.
+          - If `excitation`, vertical emittance is the result of an excitation
+            (e.g. from a feedback system) and is determined from the horizontal
+            emittance based on the value of `emittance_coupling_factor`. In this
+            case the total transverse emittance is NOT preserved.
+        This has no effect by default as `emittance_coupling_factor` defaults to 0.
+    overwrite_sigma_zeta : float, optional
+        The RMS bunch length. If provided, overwrites the one computed from
+        the longitudinal emittance. Defaults to `None`.
+    overwrite_sigma_delta : float, optional
+        The RMS momentum spread of the bunch. If provided, overwrites the one
+        computed from the longitudinal emittance. Defaults to `None`.
     rtol : float, optional
-        Relative tolerance for equilibrium emittance convergence.
-        Default is 1e-6.
+        Relative tolerance to determine when convergence is reached: if the relative
+        difference between the computed emittances and those at the previous step is
+        below `rtol`, then convergence is considered achieved. Defaults to 1e-6.
+    **kwargs : dict
+        Keyword arguments are passed to the growth rates computation method of
+        the chosen IBS formalism implementation. See the formalism classes in
+        the ``xfields.ibs._analytical`` for more details.
```

Comment:
Since I made `kwargs` passed along to the private function, I include this info here too.